cluster_tools/cluster_tools/remote.py

```diff
@@ -78,10 +78,10 @@ def worker(executor, workerid, job_array_index, job_array_index_offset, cfut_dir
         out = pickling.dumps(result)
 
     except Exception:
-        print(traceback.format_exc())
 
         result = False, format_remote_exc()
-        logging.warning("Job computation failed.")
+        logging.warning("Job computation failed with:")
+        print(traceback.format_exc())
```

Comment:
I'm wondering why this is partially in logging.warning and in print? Maybe we should rather put the traceback also in the logging statement?

Reply:
good point. i cleared it up now :ok_hand: